croud/config.py

```diff
@@ -22,35 +22,59 @@
 import yaml
 from appdirs import user_config_dir
 
+from croud.util import nested_dict_value
+
 
 class Configuration:
     USER_CONFIG_DIR: str = user_config_dir("Crate")
     FILENAME: str = "croud.yaml"
     FILEPATH: str = f"{USER_CONFIG_DIR}/{FILENAME}"
+    DEFAULT_CONFIG: dict = {
+        "auth": {"currentContext": 0, "contexts": [{"env": "prod", "token": ""}]}
```

Comment:
Could you use snake_case over camelCase please :)
```suggestion
        "auth": {"current_context": 0, "contexts": [{"env": "prod", "token": ""}]}
```